dydx/PodFile

```diff
@@ -84,7 +84,7 @@ end
 def swifiui_pods
   pod 'SDWebImage'
   pod 'SDWebImageSVGCoder'
-  pod 'SDWebImageSwiftUI'
+  pod 'SDWebImageSwiftUI', '2.2.6'
```

Comment:
Fix this version for now, as the new version has a breaking change: https://linear.app/dydx/issue/MOB-330/ios-upgrade-sdwebimageswiftui-pod